src/client/components/CustomPackCard.tsx

```diff
@@ -107,7 +107,7 @@ const CustomPackCard: React.FC<CustomPackCardProps> = ({
                 className="me-2"
                 color="accent"
                 onClick={() => {
-                  setPack({ ...pack, slots: [...pack.slots, ''] });
+                  setPack({ ...pack, slots: [...pack.slots, '*'] });
```

Comment:
Quality of life improvement, you don't have to edit the new input. From testing you can save correctly with the input blank and I am guessing on the backend that is equivalent to *